console.py

```
@@ -46,10 +46,10 @@ def _key_value_parser(self, args):
                 else:
                     try:
                         value = int(value)
-                    except:
+                    except Exception:
```

Fixes related to PEP8/pycodestyle(E722). Do not use bare except